### PR TITLE
Scrum 5 get all calls

### PR DESCRIPTION
**Pull request by AlisonGuzman2006**
Help me with this pull request please.

### CODE REVIEW

**Review by Jasonn5 — DISMISSED** [review]
All test cases were executed, and they ran as expected, there are no observations in the code, approved

File: config.py
Comment (Diego-Claros17) [inline config.py:1]: GJ

File: src/assertions/assertion_response_data.py
Comment (Diego-Claros17) [inline src/assertions/assertion_response_data.py:1]: Clean code, no objections

File: src/assertions/assertion_schemas.py
Comment (Diego-Claros17) [inline src/assertions/assertion_schemas.py:1]: No problem for me

File: tests/calls/test_getCalls.py
Comment (Diego-Claros17) [inline tests/calls/test_getCalls.py:1]: Test are ok and run as expected

**Review by Jasonn5 — APPROVED** [review]
All change are necessary, all tests were executed successfully, good job

**Review by Diego-Claros17 — APPROVED** [review]
All the added code works correctly and does not affect the rest of the code, the tests run as expected and the guidelines of the structure in which we are working are followed, which is why I accept the merge without any objection. Well done Ali.

File: config.py
Comment (ErickaIsa) [inline config.py:1]: Done

File: tests/calls/test_getCalls.py
Comment (ErickaIsa) [inline tests/calls/test_getCalls.py:1]: done

File: src/assertions/assertion_response_data.py
Comment (ErickaIsa) [inline src/assertions/assertion_response_data.py:1]: done

File: src/assertions/assertion_schemas.py
Comment (ErickaIsa) [inline src/assertions/assertion_schemas.py:1]: done

**Review by ErickaIsa — APPROVED** [review]
The changes have been reviewed and are approved. They will be committed to the develop branch.

**Pull request merged**